src/components/Authentication.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { Button, Icon, Segment, Form } from "semantic-ui-react";
+import axios from "axios"
+
+const Authentication = () => {
+  const dispatch = useDispatch();
+  const performAuthentication = async (event) => {
+    event.preventDefault();
+    let credentials = {
+      email: event.target.email.value,
+      password: event.target.password.value,
+    };
+    let response = await axios.post("/auth/sign_in", credentials);
```

Comment:
you are still not using J-tockauth here...

Reply:
We refactored to use J-tockauth now.